packages/viewer/src/interactions/interactionApi.ts

```diff
@@ -1,22 +1,24 @@
-import { Scene, Camera } from '@vertexvis/graphics3d';
+import { Camera, Scene } from '@vertexvis/graphics3d';
 import { Dimensions, Point, Vector3 } from '@vertexvis/geometry';
 import { EventEmitter } from '@stencil/core';
 import { TapEventDetails, TapEventKeys } from './tapEventDetails';
 import { FrameStreamingClient } from '../frame-streaming-client';
+import { vertexvis } from '@vertexvis/frame-stream-protos';
```

Comment:
very small nit: is this where the linter puts this import? would expect it to be with others from `@vertexvis`. also may not be used?

Reply:
It does look like this is the case, thinking it potentially has to do with the fact that `vertexvis` is lowercase, but the linter prefers to put this import toward the bottom of the list